Library/Formula/elm.rb

```diff
@@ -0,0 +1,23 @@
+class Elm < Formula
+  homepage "http://elm-lang.org"
+  url "https://github.com/kevva/elm-bin/raw/v1.4.1/vendor/osx/elm-platform-osx.tar.gz"
```

Comment:
It's the official [`npm`](http://npmjs.com) repo: https://www.npmjs.com/package/elm

@evancz can presumably vouch for it, if need be :smiley: